app/components/clock/Clock.jsx

```diff
@@ -33,7 +54,15 @@ class Clock extends React.Component {
     render() {
         return (
             <span className="clock">
-              {this.state.dateNow}
+            <span className="hours">{this.state.hours}:</span>
+            <span className="minutes">{this.state.minutes}:</span>
+            <span className="seconds">{this.state.seconds} </span>
```

Comment:
In case if `hourFormat` is `24` this space in the tail of *seconds* will be redundant.